tests/unit/cluster/replica-migration.tcl

```diff
@@ -157,3 +157,38 @@ start_cluster 4 4 {tags {external:skip cluster} overrides {cluster-node-timeout
         }
     }
 } my_slot_allocation cluster_allocate_replicas ;# start_cluster
+
+start_cluster 4 4 {tags {external:skip cluster} overrides {cluster-node-timeout 1000 cluster-migration-barrier 999}} {
+    test "valkey-cli make source node ignores NOREPLICAS error when doing the last CLUSTER SETSLOT" {
+        # Primary disable the replica-migration and replica enable the replica-migration.
```

Comment:
we can probably skip this comment. the code is self-explanatory 
```suggestion
```

Reply:
i was hoping make a tip that can show R 3 is a primary and R 7 is a replica, but you are right, the code is self-explanatory enough